app/src/main/java/ca/ualberta/CMPUT3012019T02/alexandria/activity/viewuserbook/ViewAcceptedUserBook.java

```diff
@@ -0,0 +1,28 @@
+package ca.ualberta.CMPUT3012019T02.alexandria.activity.viewuserbook;
+
+import android.os.Bundle;
+import android.view.View;
+
+import ca.ualberta.CMPUT3012019T02.alexandria.R;
+
+public class ViewAcceptedUserBook extends ViewRequestedUserBook {

```

Comment:
What naming convention are we using? We already have MainActivty this uses View. I think <Name>Activity is standard in most android projects

Reply:
Changed it to <Name>Activity